### PR TITLE
(tlg0007.tlg068.perseus-grc1.xml) EpiDoc and CTS conversion

### LINKED ISSUE #583: (tlg0007.tlg068.perseus-grc1.xml) EpiDoc and CTS conversion
Claiming file for conversion work

### PR DESCRIPTION
**Pull request by kcooper42**
fix #583 
Edited file for EpiDoc and CTS compliance

### CODE REVIEW

File: data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml [hunk 1]
```diff
@@ -0,0 +1,2231 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+<fileDesc>
+<titleStmt>
+<title xml:lang="lat">Quomodo adolescens poetas audire debeat</title>
```
Comment (lcerrato) [inline data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml]: The title appears in Greek in the book, so we should use that title.

File: data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml [hunk 2]
```diff
@@ -0,0 +1,2231 @@
...
+<name>David Smith</name>
+</respStmt>
+<funder n="org:NEH">The National Endowment for the Humanities</funder>
+</titleStmt>
+<!--<extent/>-->
+<publicationStmt>
+<publisher>Trustees of Tufts University</publisher>
+<pubPlace>Medford, MA</pubPlace>
+<authority>Perseus Project</authority>
+<idno type="filename">tlg0007.tlg068.perseus-grc2.xml</idno>
```
Comment (lcerrato) [inline data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml]: We do not typically use this tag. Let's strike it.

File: data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml [hunk 3]
```diff
@@ -0,0 +1,2231 @@
...
+</publicationStmt>
+<sourceDesc>
+<biblStruct>
+<!--<analytic/>-->
+<monogr>
+<author ref="urn:cts:greekLit:tlg0007">Plutarch</author>
```
Comment (lcerrato) [inline data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml]: we don't typically add the author ref here

File: data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml [hunk 4]
```diff
@@ -0,0 +1,2231 @@
...
+<!--<title type="sub">Machine readable text</title>-->
+<author>Plutarch</author>
+<editor role="editor" n="Teubner">Gregorius N.
+            Bernardakis</editor>
+<sponsor>Perseus Project, Tufts University</sponsor>
+<principal>Gregory Crane</principal>
+<respStmt>
+<resp>Prepared under the supervision of</resp>
+<name>Lisa Cerrato</name>
+<name>William Merrill</name>
```
Comment (lcerrato) [inline data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml]: Please remove the names Merrill, Smith and Mylonas. Please add the names Rashmi Singhal and Bridget Almas. (This is something I need to edit and saves me a PR.) Thanks!

File: data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml (hunk 2 above)
Comment (lcerrato) [reply data/tlg0007/tlg068/tlg0007.tlg068.perseus-grc2.xml]: Please add `<date type="release">2010-12-13'</date>`. This is something that saves me a new PR. Thanks.

**Comment by kcooper42** [comment]
@lcerrato I've updated the file to correspond with your suggested changes.

**Comment by lcerrato** [comment]
Great job! Thank you, @kcooper42.

**Comment by lcerrato** [comment]
One last thing, please delete the base files:
tlg0007.tlg068.perseus-grc1.tracking.json
and
tlg0007.tlg068.perseus-grc1.xml
and add to this pull request.

**Comment by kcooper42** [comment]
@lcerrato Okay, I've deleted those files. For future reference, should we delete these base files before the initial pull request or wait until it has been approved first? Thanks!

**Comment by lcerrato** [comment]
@kcooper42 
Good question. It honestly does not matter much. I can always clean them up later, but I would say once I see that the new file passes all the checks, it is a good idea to delete the old file and the matching tracking file.

I like to keep the old file around until the checks are ok.

It is really an afterthought (I'm not sure it's even in my instructions — I'll check now.)

**Pull request merged**